style.css

```diff
@@ -675,21 +715,26 @@ input[type=text], select, textarea {
 	border: 1px solid #ccc;
 	border-radius: 5px;
 	resize: vertical;
+	font-size: 14px;
+    font-family: 'Telex', sans-serif;
 }
 
 input[type=submit] {
-	background-color: #478ac9;
+	background: #0080FF;
 	color: white;
 	padding: 12px 20px;
 	border: none;
 	border-radius: 4px;
 	cursor: pointer;
 	float: right;
 	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    font-family: 'Telex', sans-serif;

```

Comment:
Font-styling and a size of 14px was added as I forgot to do this previously in design 1